pkgs/top-level/all-packages.nix

```diff
@@ -6493,7 +6493,7 @@ in
 
   tab = callPackage ../tools/text/tab { };
 
-  tautulli = callPackage ../servers/tautulli { python = python2; };
+  tautulli = callPackage ../servers/tautulli { python = python3; };
```

Comment:
Can we just pass in python3?

Reply:
I don't see why not. I will change the PR accordingly